CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.6.0 (TBD)
+## 1.6.0 (2023-07-28)
```

Comment:
Line 14 seems to have a wrong indentation

Reply:
Are we waiting for the connector to be released first?

Reply:
> Line 14 seems to have a wrong indentation

It is a sub-bullet point for the auto-upload feature

Reply:
Oh okay 🤦 I guess I need a break lmao

Reply:
You might need to update the date from 28 to 31.

Reply:
I don't know why, but I can't add feedback to specific lines below this point. Maybe the content was added in a different PR or something? Anyway, here is my feedback about the rest of the new changelog content:

Change: 
```
- Added support for new functions in `snowflake.snowpark.functions`:
```
to 
```
"- Added support for these new functions in `snowflake.snowpark.functions`:"
```

Reply:
Change 
```
- Added support for pure Python packages specified via `Session.add_requirements` or `Session.add_packages`, now usable in stored procedures and UDFs even if packages are not present on the Snowflake Anaconda channel.
```

to

```
- Added support for pure Python packages specified via `Session.add_requirements` or `Session.add_packages`. They are now usable in stored procedures and UDFs even if packages are not present on the Snowflake Anaconda channel.
```

Reply:
Change

```
- Added support for async execution of multi-query dataframe containing binding variables.
```

to

```
- Added support for asynchronous execution of multi-query dataframes that contain binding variables.
```

Reply:
Regarding this line, I'm not sure you should mention private preview features. If you do decide to include it, please spell out what UDAF stands for like this "user-defined aggregate functions (UDAFs) "

```
- Added support for UDAF. This feature is currently in private preview.
```

Reply:
SQL should be capitalized in this sentence:

```
- Fixed a bug where `DataFrame.union` was not generating the correct `schema_query` when sql simplifier is enabled.
```

Reply:
Hi Mary, I'm addressing these comments, for the last one, I don't think `schema_query` is a SQL term here. cc: @sfc-gh-aalam

Reply:
@sfc-gh-stan Sorry, I think I wasn't clear in my comment. I'm not suggesting that you capitalize "schema_query". I'm just saying that when we talk about the SQL language in general, the name of the language, SQL, should be capitalized. So the sentence would be ..."...when SQL simplifier...".

Reply:
Gotcha, thanks Mary!